data/init.sql

```diff
@@ -36,3 +36,31 @@ CREATE TABLE IF NOT EXISTS discussion_messages (
 	created_at TIMESTAMP,
 	updated_at TIMESTAMP
 );
+
+CREATE TABLE IF NOT EXISTS courses (
+id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+course_code VARCHAR(60),
+course_name VARCHAR(60),
+created_at TIMESTAMP,
+updated_at TIMESTAMP
+); 
+
+CREATE TABLE IF NOT EXISTS course_offerings(
+id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+course_id INT(11) UNSIGNED,
+FOREIGN KEY(course_id) REFERENCES courses(id) ON DELETE CASCADE,
+course_offering_code VARCHAR(60),
+course_offering_name VARCHAR(60),
```

Comment:
Isn't the course offering a semester?  Dont we need a semester argument?

Reply:
oh I thought we didnt want to introduce the notion of a semester?

Reply:
I think I'm just unclear on what the offering is then? What does it represent?

Reply:
no i think you've got the right idea of what an offering is, but I remember us talking about how the semester should just be something that people can add in the name of a course offering